Topics/lifecycle.md

```diff
@@ -0,0 +1,111 @@
+
+If we want the data to work for us, we need to be able to ask the right questions. Only then will the data be able to offer us broad perspectives, make generally accurate forecasts and increase our knowledge.
+
+In addition, Problem Shaping is a classic "self-generating" process, similar to brainstorming: good and innovative questions help the team to generate more intelligent questions. At this stage of the project, lateral thinking is a very valuable soft skill.  
+
+Another significant aspect is the ability to transmit the results provided by the data. Human beings, by their very nature, have prejudices that influence the way they perceive the results, so it is necessary to find the most effective way to "tell the story".
+history" of the data, a very important step for the success of a project.
+
+### Tools that can help in the definition of the problem
+
+The definition of the problem is the phase of the Data Science process that more than any other depends on soft skills (compared to technological skills or hard skills), however, since it is based on questions and data, sometimes in large quantities, can be
+A data analysis tool is useful.
+
+At this stage of the project, a key factor is the collaboration between data scientists and business users who, ultimately, are those with the most extensive knowledge in the field and are therefore those who will pave the way for success. From our experience, this collaboration is facilitated by data visualization tools. 
+
+Data visualization tools such as [Tableau](https://www.tableau.com/it-it) generally have direct access to various types of structured and unstructured data sources, so that they can be applied to raw data, and are extremely effective in identifying trends, anomalies or aberrant data in the analyzed data with a much higher level of productivity than the classic tabular approach.  
+
+For simple views, the Python [MatPlotLib](https://matplotlib.org/) library does a great job. We will see in a later guide how to use it.
+
+As mentioned above, we must bear in mind that a Data Science project must always be geared towards achieving results focused on the activity and have a global vision in line with the business strategy.
+
+In this regard, I would like to point out the project concept [S.M.A.R.T.](https://www.getorganizedwizard.com/smart-goals/an-overview-of-smart-project-management/), which helps to frame the problem and encourages wise evaluations during this phase.
+
+Machine learning projects must have short-term objectives and be managed with an agile approach: the recirculation of questions, hypotheses and related data must be continuous, new discoveries must be used to guide and improve subsequent series of projects and results, even partial, must be shared with stakeholders (the business, colleagues, or simply those interested).
```

Comment:
See now you are talking about ML project, in the Project called DS Project - even though I know the topic i'm thinking it might be ambiguous to a newbie!

Reply:
True thanks